pybop/_problem.py

```diff
@@ -259,6 +259,12 @@ def __init__(
                 init_soc=self.init_soc,
             )
 
+        # Add an example dataset for plotting comparison
```

Comment:
I'm not sure I follow this addition. Could this be completed outside of the `DesignProblem` class, maybe by grabbing the first step in the optimiser?

Reply:
Possibly? It could be left to the user to add these steps if desired. I added them here simply to ensure that there is a target dataset for the plotting comparisons.